scripts/static-styles/config.js

```diff
@@ -17,22 +17,22 @@ import React from 'react';
 import { entries, isFunction, kebabCase } from 'lodash/fp';
 import { light } from '@sumup/design-tokens';
 
-import {
-  // Badge,
-  ButtonGroup,
-  Card,
-  Checkbox,
-  Hr,
-  Image,
-  // Input,
-  // Label,
-  RadioButton,
-  // Select,
-  // Selector,
-  Tag,
-  // TextArea,
-  Toggle
-} from '../../src';
+// import {
```

Comment:
What's the purpose of commenting those instead of removing them?

Reply:
See https://github.com/sumup-oss/circuit-ui/pull/615#discussion_r443348369 :)

Reply:
Ah now it makes sense 👍